Common/UnitDefinitions/ElectricReactiveEnergy.json

```diff
@@ -0,0 +1,25 @@
+{
+  "Name": "ElectricReactiveEnergy",
+  "BaseUnit": "VoltampereReactiveHour",
+  "XmlDocSummary": "The volt-ampere reactive hour (expressed as varh) is the reactive power of one Volt-ampere reactive produced in one hour.",
+  "BaseDimensions": {
+    "L": 2,
+    "M": 1,
+    "T": -1
```

Comment:
I maybe wrong here, but shouldn't this be `"T": -2`? I have it changed in one of my uncommitted stashes from months ago (back then it was still called `ReactiveEnergy` 😄 ).

Reply:
Looks like you are correct. I'll make the change right now, in both ReactiveEnergy and ElectricReactiveEnergy, and regerate the code.

Thanks!